resdk/resources/data.py

```diff
@@ -307,13 +307,16 @@ def stdout(self):
         :rtype: string
 
         """
+        # Prevent circular imports:
+        from resdk.resolwe import CHUNK_SIZE
```

Comment:
Move `CHUNK_SIZE` to `constants.py` (or similar), so there will be no circular import problems.